app/views/pages/home.html.erb

```diff
@@ -1,4 +1,34 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
-<p>Hello its antonio</p>
-<p>Hello, it's Mariana</p>
+<div class="background">
+  <div class="container">
+    <div class="name">
+      <h3>Welcome to</h3>
+      <h1>Rentart.</h1>
+      <h3>Do you want to rent the Mona Lisa for a day?</h3>
+      <div class="link-to-index">
+        <p><%= link_to "Book a piece of art >>", root_path, class:"text-decoration-none" %></p>
+      </div>
+    </div>
+    <div class="image">
+      <img src="https://i.pinimg.com/564x/46/cc/ae/46ccae735509fe7ff3284639c781b9ef.jpg" alt="Polock painting" width="375px">
+    </div>
+  </div>
+  <div class="sign-in">
+    <h3>Not registered yet?</h3>
+    <p>render form</p>
+    <img src="https://d2x3xhvgiqkx42.cloudfront.net/12345678-1234-1234-1234-1234567890ab/9432a2ad-f01d-4a3d-ae53-370c37e15e62/2019/04/03/d43fd77a-7eff-44a0-bddf-f3314c60e521/7103b8d3-eadc-40c3-bc6c-283fece6746b.png" alt="Polock painting" width="375px"></div>
+  </div>
+  <div class="categories">
+    <h3>Find by category</h3>
+    <div class="category-cards">
+      <div class="card-category">
+        <p>Paintings</p>
+      </div>
+      <div class="card-category">
+        <p>Sculptures</p>
+      </div>
+      <div class="card-category">
+        <p>Instalations</p>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
The whole file is very dense. Can you add some spacing between sibling elements to make it easier to read ?